session-3/TaskManagementSystem/main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	_   = 0
+	low = iota
+	medium
+	high
+	tasksAvailable int = 100
+)
+
+func main() {
+	var (
+		projectStatus   = "IN PROGRESS"
+		tasksCompleted  = 100
+		isCompleted     = false
+		tasksRemaining  = tasksAvailable - tasksCompleted
+		projectProgress = ""
+	)
+
+	if tasksCompleted == tasksAvailable {
+		isCompleted = true
+	}
+
+	if tasksRemaining == 0 {
+		projectStatus = "FINISHED"
+	}
+
+	switch {
+	case tasksCompleted < 30:
+		projectProgress = "Starting phase"
+	case tasksCompleted >= 30 && tasksCompleted < 60:
```

Comment:
in case of equality to 60, there will be lack of cases. I propose to add equal to 60 on that case
```suggestion
	case tasksCompleted >= 30 && tasksCompleted <= 60:
```